pkgs/development/python-modules/keyutils/default.nix

```
@@ -14,9 +14,10 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace setup.py --replace '"pytest-runner"' ""
+    ${if pythonAtLeast "3.11" then "${cython}/bin/cython keyutils/_keyutils.pyx" else ""}
```

Let's just ddo it for all Python versions. There's no harm in doing so.
```suggestion
  '';
  
  postConfigure = ''
    cython keyutils/_keyutils.pyx
```